igvc_platform/src/motor_controller/motor_controller.cpp

```diff
@@ -168,101 +164,102 @@ void MotorController::setPID()
-    uint8_t buffer[256];  // buffer to read response into
+  /* Read response from the server */
+  size_t n;             // n is the response from socket: 0 means connection closed, otherwise n = num bytes read
+  uint8_t buffer[256];  // buffer to read response into
 
-    memset(buffer, 0, sizeof(buffer));
-    /* read from the buffer */
-    n = (*sock_).readMessage(buffer);  // blocks until data is read
+  memset(buffer, 0, sizeof(buffer));
+  /* read from the buffer */
+  n = (*sock_).readMessage(buffer);  // blocks until data is read
 
-    if (n == 0)
-    {
-      ROS_ERROR_STREAM("Connection closed by server");
-      ros::shutdown();
-    }
+  if (n == 0)
+  {
+    ROS_ERROR_STREAM("Connection closed by server");
+    ros::shutdown();
+  }
 
-    /* Allocate space for the decoded message. */
-    ResponseMessage response = ResponseMessage_init_zero;
+  /* Allocate space for the decoded message. */
+  ResponseMessage response = ResponseMessage_init_zero;
 
-    /* Create a stream that reads from the buffer. */
-    pb_istream_t istream = pb_istream_from_buffer(buffer, n);
+  /* Create a stream that reads from the buffer. */
+  pb_istream_t istream = pb_istream_from_buffer(buffer, n);
 
-    /* decode the message. */
-    bool status = pb_decode(&istream, ResponseMessage_fields, &response);
+  /* decode the message. */
+  bool status = pb_decode(&istream, ResponseMessage_fields, &response);
 
-    /* check for any errors.. */
-    if (!status)
-    {
-      ROS_ERROR_STREAM("Decoding Failed: " << PB_GET_ERROR(&istream));
-      ros::shutdown();
-    }
+  /* check for any errors.. */
+  if (!status)
+  {
+    ROS_ERROR_STREAM("Decoding Failed: " << PB_GET_ERROR(&istream));
+    ros::shutdown();
+  }
 
-    /* update the exponentially weighted moving voltage average and publish */
-    std_msgs::Float64 battery_msg;
-    battery_avg_ = battery_alpha_ * battery_avg_ + (1 - battery_alpha_) * response.voltage;
-    battery_msg.data = battery_avg_;
-    battery_pub_.publish(battery_msg);
+  /* update the exponentially weighted moving voltage average and publish */
+  std_msgs::Float64 battery_msg;
+  battery_avg_ = (battery_alpha_ * battery_avg_) + ((1 - battery_alpha_) * response.voltage);
+  battery_msg.data = battery_avg_;
+  battery_pub_.publish(battery_msg);
 
-    if (battery_avg_ < min_battery_voltage_)
-    {
-      ROS_ERROR_STREAM_THROTTLE(log_period_, "Battery voltage dangerously low:"
-                                                << "\n\tCurr. Voltage: " << battery_avg_
-                                                << "\n\tMin. Voltage: " << min_battery_voltage_);
-    }
+  if (battery_avg_ < min_battery_voltage_)
+  {
+    ROS_ERROR_STREAM_THROTTLE(log_period_, "Battery voltage dangerously low:"
+                                               << "\n\tCurr. Voltage: " << battery_avg_
+                                               << "\n\tMin. Voltage: " << min_battery_voltage_);
+  }
```

Comment:
This could probably also be refactored into a separate method for visual clarity

Reply:
addressed